source/common/http/utility.h

```diff
@@ -106,6 +106,8 @@ struct OptionsLimits {
   static const uint32_t DEFAULT_MAX_INBOUND_PRIORITY_FRAMES_PER_STREAM = 100;
   // Default limit on the number of inbound frames of type WINDOW_UPDATE (per DATA frame sent).
   static const uint32_t DEFAULT_MAX_INBOUND_WINDOW_UPDATE_FRAMES_PER_DATA_FRAME_SENT = 10;
+  // defaults to unlimited, value in octet
+  static const uint32_t DEFAULT_MAX_HEADER_LIST_SIZE = std::numeric_limits<uint32_t>::max();
```

Comment:
Envoy will by default reject request headers block larger than 60 KiB (`DEFAULT_MAX_REQUEST_HEADERS_KB`), so we shouldn't be advertising 4 GiB limit, leading peers to believe otherwise.

Reply:
default to 60KiB is fine for this?